libs/remix-debug/src/Ethdebugger.ts

```diff
@@ -100,6 +101,27 @@ export class Ethdebugger {
     return this.callTree.findScope(step)
   }
 
+  async decodeLocalVariableByIdAtCurrentStep (step: number, id: number) {
+    const variable = this.callTree.getLocalVariableById(id)
+    if (!variable) return null
+    const stack = this.traceManager.getStackAt(step)
```

Comment:
this fails when restarting the debugger via the debugger itself. the trace is empty, if you start it with the terminal it's fine.